lua/nvim-treesitter/parsers.lua

```diff
@@ -241,7 +247,8 @@ list.typescript = {
   install_info = {
     url = "https://github.com/tree-sitter/tree-sitter-typescript",
     files = { "src/parser.c", "src/scanner.c" },
-    location = "tree-sitter-typescript/typescript"
+    location = "tree-sitter-typescript/typescript",
+    generate_requires_npm = true,
```

Comment:
typescript is on 0.19 now

Reply:
Bout generate requires still npm if you want to Do it